Makefile

```diff
@@ -265,3 +265,19 @@ crc-e2e: crc-instance ${E2E_TEST}
 	gcloud compute ssh crc@${CRC_INSTANCE_NAME} -- rm -f ./e2e
 	gcloud compute scp --compress ${E2E_TEST} crc@${CRC_INSTANCE_NAME}:e2e
 	gcloud compute ssh crc@${CRC_INSTANCE_NAME} -- ./e2e --repo-root=/dev/null --ginkgo.focus="CA\ Issuer" --ginkgo.skip="Gateway"
+
+
+
+.PHONY: update-community-operators
+update-community-operators: export UPSTREAM := k8s-operatorhub
+update-community-operators: export FORK := wallrj
```

Comment:
In another PR I'll try and set up forks in the github.com/cert-manager org so that these PRs can come from a more official source.